mailpoet/assets/img/icons/warning.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="-2 -2 24 24">
```

Comment:
You mentioned this is included in `@wordpress/icons`. In that case, we don't need the SVG, do we? Or did you want to use the image from Figma?

Reply:
Aaaah, that's due to the usage in CSS? In that case, if we need an SVG anyway, I guess we could use the icon from Figma, maybe.

Reply:
Yes. For me the question is more, do we align with `@wordpress/icons` or Figma at that point from I guess a product perspective. @NeosinneR related Slack was here: https://a8c.slack.com/archives/C0313447YBC/p1666332588178519

Reply:
@websupporter I like @JanJakes suggestion that if we need to add the SVG into codebase anyway, let's stick to Figma. But if we can use the existing `@wordpress/icons`, let's choose that.

Reply:
Addressed in [e5b6aec](https://github.com/mailpoet/mailpoet/pull/4447/commits/e5b6aec4beca57082c8102a0e87434beca44e18e)